deployment/kubernetes/charts/origin/templates/eth.ingress.yaml

```diff
@@ -13,6 +13,7 @@ metadata:
     certmanager.k8s.io/cluster-issuer: {{ .Values.clusterIssuer }}
     nginx.ingress.kubernetes.io/rewrite-target: /
     nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+    nginx.ingress.kubernetes.io/limit-rps: "2"
```

Comment:
That seems low. The DApp when not using discovery server makes more than 1 web3 call per listing card on browse/search page. I'd set this to something in the order of 24.